_posts/2017/2017-11-01-internal-workshops.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "Gradle Workshop"
```

Comment:
Change to something like:

"Gradle or not Gradle, that is the question" (à la Hamlet :D)
"Why (not) using Gradle?"

Reply:
Have that as a chapter title already...

Reply:
Let's just keep it like that, I have thought long enough about it...